### DISCUSSION TITLE
Help with calendar component

### DISCUSSION THREAD

**Discussion by crnikaurin** [Budibase/budibase#10063] [Help] [answered]
Hi. I need some help with calendar component with steps to connect data provider and date/time mapping source. If I only connect data provider, I assume that in further steps need to extract date/time parameter from json object (how to do that?

Regards

Danijel

**Comment by ConorWebb96** [comment, marked as answer]
Hey @crnikaurin,

Are you referring to this plugin? https://github.com/marblekirby/budibase-calendar-plugin ?

Reading the description it should just be as simple as installing the plugin. Then declaring the Title and Date field used within the  specific table you are using. (The plugin will likely handle the rest from there.)

If there are any issues which arise that are related to this plugin you will need to get in contact with the maintainer, his Github can be found here: https://github.com/marblekirby

Hopefully this helps!
